earthpy/tests/test_spatial.py

```diff
@@ -48,44 +41,39 @@ def test_extent_to_json():
     with pytest.raises(AssertionError):
         es.extent_to_json([0, 1, 1, 0])
 
+
 def test_bytescale_high_low_val():
     """"Unit tests for earthpy.spatial.bytescale """
-    
-    arr = np.random.randint(300, size=(10,10))
-    
+
+    arr = np.random.randint(300, size=(10, 10))
+
     # Bad high value
     with pytest.raises(ValueError):
         es.bytescale(arr, high=300)
-        
+
     # Bad low value
     with pytest.raises(ValueError):
         es.bytescale(arr, low=-100)
-        
+
     # High value is less than low value
     with pytest.raises(ValueError):
         es.bytescale(arr, high=100, low=150)
-        
+
     # Valid case. should also take care of if statements for cmin/cmax
     val_arr = es.bytescale(arr, high=255, low=0)
-        
+
     assert val_arr.min() == 0
     assert val_arr.max() == 255
-    
+
     # Test scale value max is less than min
     with pytest.raises(ValueError):
         es.bytescale(arr, cmin=100, cmax=50)
-        
-        
+
     # Test scale value max is less equal to min
     es.bytescale(arr, cmin=100, cmax=100)
-    # assert something    
-        
+    # TODO: add assertion here
```

Comment:
Yes... so we had a bytescale snafu today. and Joe commented this out completely but then we realized it wasn't actually the cause of the issue at all.. so you're code here might be just fine. I suspect travis was going wonky.

Reply:
OK - I commented out this line to go with the most recent version, and left the TODO part to remind us to write a real test for this case.